### PR TITLE
View for multiple host tabs added

### PR DESCRIPTION
**Pull request by LadislavVasina1**
In this PR, I am improving airgun by adding new host details cards and finishing unfinished views (Overview, Details, Content, Parameters, Traces, Reports, Insights, Puppet). Basic entities for reading values from UI are present too.

For more info see SAT-16452.

### CODE REVIEW

File: airgun/entities/host_new.py [hunk 1]
```diff
@@ -178,6 +178,142 @@ def get_tracer(self, entity_name):
...
+        net_devices = [
+            i.split()[0]
+            for i in view.details.networking_interfaces.networking_interfaces_accordion.items()
+        ]
+
+        for dev in net_devices[1:]:
+            view.details.networking_interfaces.networking_interfaces_accordion.toggle(dev)
+
+        def val_gen(item_name):
+            """Generator for values in the networking interfaces accordion"""
```
Comment (pondrejk) [inline airgun/entities/host_new.py:237]: here the name sounds a bit misleading, does it really generate values?

**Review by pondrejk — APPROVED** [review]
overall ack

File: airgun/entities/host_new.py (hunk 1 above)
Comment (LadislavVasina1) [reply airgun/entities/host_new.py:237]: for the dictionary yes

Comment (LadislavVasina1) [reply airgun/entities/host_new.py:237]: but not in the way that they are not in the UI. I needed to generate them one by one for each interface that is saved in the dictionary

Comment (LadislavVasina1) [reply airgun/entities/host_new.py:237]: I can change the comment

**Comment by pnovotny** [comment]
Views for the host Details cards have been verified here: https://github.com/SatelliteQE/robottelo/pull/11133/
The test is just for the Provisioning Templates card, however the `NewHostEntity.get_details()` function reads all the cards on the Details tab.

**Review by vsedmik — APPROVED** [review]
Looks good to me

**Pull request merged**